test/ReactWrapper-spec.js

```diff
@@ -1632,7 +1632,44 @@ describeWithDOM('mount', () => {
     });
   });
 
-  describe('.html()', () => {
+  describeIf(REACT15, '.html()', () => {
+    it('should return html of straight DOM elements', () => {
+      const wrapper = mount(
+        <div className="test">
+          <span>Hello World!</span>
+        </div>
+      );
+      expect(wrapper.html()).to.equal(
+        `<div data-reactroot="" class="test"><span>Hello World!</span></div>`
+      );
+    });
+
+    it('should render out nested composite components', () => {
+      class Foo extends React.Component {
+        render() {
+          return (<div className="in-foo" />);
+        }
+      }
+      class Bar extends React.Component {
+        render() {
+          return (
+            <div className="in-bar">
+              <Foo />
+            </div>
+          );
+        }
+      }
+      const wrapper = mount(<Bar />);
+      expect(wrapper.html()).to.equal(
+        `<div data-reactroot="" class="in-bar"><div class="in-foo"></div></div>`
```

Comment:
similar to how we used to remove the `data-reactid` attributes, we should maybe also remove the `data-reactroot` attribute as well. Makes the return value of this function more consistent actually, and can prevent the need for us to make these tests different per version